flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -433,8 +427,7 @@ public void testBatchQueryExecutionMultipleTimes() throws Exception {
     }
 
     @Test
-    @Timeout(value = 90)
-    public void testStopJob() throws Exception {
+    void testStopJob() throws Exception {
```

Comment:
The line in this method `assertThat(savepoint.isPresent()).isTrue();` could be simplified as
`assertThat(savepoint).isPresent();`